docs/_plugins/copy_api_dirs.rb

```diff
@@ -26,8 +26,8 @@
     curr_dir = pwd
     cd("..")
 
-    puts "Running 'build/sbt -Pkinesis-asl clean compile unidoc' from " + pwd + "; this may take a few minutes..."
-    system("build/sbt -Pkinesis-asl clean compile unidoc") || raise("Unidoc generation failed")
+    puts "Running 'build/sbt -Pscala-2.13 -Pkinesis-asl clean compile unidoc' from " + pwd + "; this may take a few minutes..."
```

Comment:
QQ, why do we need scala profile here?

Reply:
When we execute `dev/change-scala-version.sh 2.13`, it simultaneously modifies both `copy_api_dirs.rb` and `mima` files, adding these `-Pscala-2.13`. In this PR, I didn't clean them up. It seems we could add a follow-up to clear these unnecessary `-Pscala-2.13`.